server/src/main/java/org/apache/cassandra/sidecar/db/CdcDatabaseAccessor.java

```diff
@@ -181,31 +216,4 @@ ResultSetFuture selectCdcRange(String jobId, int split)
     {
         return session().executeAsync(tableSchema.select().bind(jobId, (short) split));
     }
-
-    public ResultSetFuture insertTableSchemaHistory(String keyspace, String tableName, String schema)
```

Comment:
Are you sure that you might not need these in the future?

Reply:
We can always add it back if we need it.